actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -202,10 +182,28 @@ def _filtered_referrer # :nodoc:
       # * <tt>:disable_with</tt> - Value of this parameter will be used as the
       #   name for a disabled version of the link. This feature is provided by
       #   the unobtrusive JavaScript driver.
+      # * <tt>remote: true</tt> - This will allow the unobtrusive JavaScript
+      #   driver to make an Ajax request to the URL in question instead of following
```

Comment:
```suggestion
      # * <tt>remote: true</tt> - This will allow @rails/ujs
      #   to make an Ajax request to the URL in question instead of following
```

It looks like you changed a couple of instances of "unobstrusive JavaScript driver" to "@rails/js" later on but left this one. I don't have a strong opinion on which one it should be but I do think it should be consistent